src/test/java/StylesCheck.java

```diff
@@ -0,0 +1,128 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.Color;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.nio.channels.WritableByteChannel;
+import java.time.Duration;
+
+public class StylesCheck {
+
+    private WebDriver driver;
+
+    @BeforeSuite
+    public void setupDriverManager() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeTest
+    public void openBrowser() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofMillis(3000));
+        driver.get("https://app.neuralegion.com/login");
+        final String email = "catbubliktest1@yandex.ru";
+        final String passwordText = "f@BzSJbGX68XAwr";
+
+        WebElement loginInput = driver.findElement(By.id("mat-input-0"));
+        WebElement passwordInput = driver.findElement(By.id("mat-input-1"));
+        loginInput.sendKeys(email);
+        passwordInput.sendKeys(passwordText);
+        WebElement submitButton = driver.findElement(By.cssSelector("auth-login-form form > button"));
+        submitButton.click();
+        new WebDriverWait(driver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.elementToBeClickable(By.cssSelector("scans-scans-toolbar button:nth-of-type(1)")));
+
+    }
+
+    @AfterTest
+    public void closeBrowser() {
+        driver.quit();
+    }
+
+    @Test
+    public void stylesCheckColorMenuButton() {
```

Comment:
I think the title of the methods are not descriptive. Consider something like `CheckMenuButtonColor`, `CheckRefreshButtonColor` etc. Here and below.